content/en/docs/languages/go/generated-code-old.md

```diff
@@ -0,0 +1,189 @@
+---
```

Comment:
Is this just a copy/paste of the old markdown file?  If so I won't review the language at all.

Reply:
Yes, it is an exact copy of the old markdown file.